packages/next/build/webpack/loaders/next-image-loader.js

```diff
@@ -11,22 +11,35 @@ function nextImageLoader(content) {
     const { isServer, isDev, assetPrefix, basePath } = this.getOptions()
     const context = this.rootContext
     const opts = { context, content }
+    const importName = loaderUtils.interpolateName(
+      this,
+      '[path][name].[ext]',
+      opts
+    )
     const interpolatedName = loaderUtils.interpolateName(
       this,
       '/static/media/[name].[hash:8].[ext]',
       opts
     )
     const outputPath = assetPrefix + '/_next' + interpolatedName
-
     let extension = loaderUtils.interpolateName(this, '[ext]', opts)
     if (extension === 'jpg') {
       extension = 'jpeg'
     }
 
     const imageSizeSpan = imageLoaderSpan.traceChild('image-size-calculation')
     const imageSize = await imageSizeSpan.traceAsyncFn(() =>
-      getImageSize(content, extension)
+      getImageSize(content, extension).catch((err) => err)
     )
+
+    if (imageSize instanceof Error) {
+      const err = new Error(
+        `Image import "./${importName}" is not a valid image format`
```

Comment:
At first sight, I won't see the issue, since the extension looks good (I understand we are looking at the actual file type, but I won't remember that when developing). Can we be more specific with the message? Something like:

"Note that the image extension might be correct, but the file type was not".

Would an error page with a link be overkill for this?